src/middlewared/middlewared/plugins/vm/supervisor/supervisor_base.py

```diff
@@ -246,6 +246,12 @@ def get_domain_children(self):
             # Command line args
             *self.commandline_xml(),
         ]
+        # Memory Ballooning - this will be memory which will always be allocated to the VM
+        # If not specified, this defaults to `memory`
+        if self.vm_data['min_memory']:
```

Comment:
Group this together with `memory` element into a separate method like `cpu_xml` please